src/css/table.css

```diff
@@ -67,3 +67,14 @@ td.sorted {
 .sort-asc.sorted:before {
   border-bottom-color: #eee;
 }
+
+.view-all-btn {
+  width: 100%;
+  padding: 1em;
+  margin: .5em 0 2.5em;
+  text-align: center;
+  font-weight: 500;
+  border: 1px #333 solid;
+  cursor: pointer;
+  user-select: none;
+}
```

Comment:
These don't quite match the design — looks like they should be right up against the bottom of the table, instead of it having some margin. It also looks like the text is bold in the design.